packages/ses/src/enablements.js

```diff
@@ -152,6 +161,10 @@ export const moderateEnablements = {
 
   '%IteratorPrototype%': {
     toString: true,
+    // https://github.com/tc39/proposal-iterator-helpers
+    constructor: true,
+    // https://github.com/tc39/proposal-iterator-helpers
+    [toStringTagSymbol]: true,
```

Comment:
Unrelated to this, but I kinda wished our enablements logic really represented "levels" and extended lower layers instead of redefining them.